__test__/work-orders/[id]/update.test.js

```diff
@@ -1,20 +1,20 @@
-import ChooseJobOptionPage from 'src/pages/repairs/jobs/[id]/close-job'
+import UpdateWorkOrderPage from 'src/pages/work-orders/[id]/update'
```

Comment:
Seem like github has got this file name confused with https://github.com/LBHackney-IT/repairs-hub-frontend/pull/167/files#diff-62687303bdc964bf8d3779b610177fdf979da50a6a3c66a7ef3302e5bb633208L1-R1